charts/s3gw/values.yaml

```diff
@@ -9,19 +9,32 @@ secret_key: "test"
 # using HTTPS and self-signed certificates because of CORS issues.
 # To workaround that, please open the URL https://<HOSTNAME> in the
 # browser and accept the SSL certificate before accessing the UI
-# via https://ui-<HOSTNAME>.
+# via https://<UI-HOSTNAME>.
 # Check https://github.com/aquarist-labs/s3gw/issues/31 to get more
 # information about the CORS issues.
 # !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 ui:
+  # 'enabled' will deploy the S3 user interface if set to `true`.
   enabled: false
+  hostname: s3gw-ui
+  hostnameNoTLS: s3gw-ui-no-tls
+  domainName: local
 
-# FQDN for the endpoint
-hostname: s3gw.local
+  # --- Developer Options ---
+  imageRegistry: "ghcr.io/aquarist-labs"
+  imagePullPolicy: "Always"
+  imageName: "s3gw-ui"
+  imageTag: "v0.4.0"
+
+# Hostname and domain name
+hostname: s3gw
+hostnameNoTLS: s3gw-no-tls
```

Comment:
Any suggestion for a better variable name?

Reply:
I'm fine with this.